LDM-148.tex

```diff
@@ -545,7 +545,7 @@ \subsection{Interfaces}\label{base-interfaces}
 from the control service to the batch service, whether to the Commissioning Cluster or over the international network to the Offline Production Enclave, and return status and
 result information.
 
-Telemetry Gateway from Prompt US Enclave services: RabbitMQ is
+Telemetry Gateway from Prompt US Enclave services: RabbitMQ or Kafka are
```

Comment:
Do we know which? I'm not sure if this “or” expresses a “TBD” or a “we use one one or the other, depending on the type of data to be transferred”.

Reply:
It's a TBD.  I could replace with a generic "messaging system", explicitly label "(TBD)", or leave as is.  Preference?

Reply:
I went with "(TBD)".